src/components/AccordionTable/AccordionTable.types.part.tsx

```diff
@@ -108,6 +108,12 @@ export interface AccordionTableProps<T> extends TableProps<T>, AccordionCardLabe
    * Notification callback if a group is expanded or collapsed.
    */
   onToggleGroup?(e: AccordionGroupToggleEvent<T>): void;
+
+  /**
+   * A label to be used for the group of those items which have no value for the 'groupBy' column.
+   * Default value is `—`.
```

Comment:
@default '-'

Reply:
with "@": `@default: '-'`

Reply:
I'm a little confused now on what the correct syntax is. Would this be correct:
```
  /**
   * A label to be used for the group of those items which have no value for the 'groupBy' column.
   * Default value is `—`.
   *
   * @default '—'
   */
```

Reply:
```
  /**
   * A label to be used for the group of those items which have no value for the 'groupBy' column.
   *
   * @default '—'
   */
```